components/builder-web/app/actions/projects.ts

```diff
@@ -120,45 +120,43 @@ export function fetchProjects(token: string) {
   };
 }
 
-export function deleteProject(id: string, token: string, origin: string) {
+export function deleteProject(id: string, token: string) {
   return dispatch => {
     new BuilderApiClient(token).deleteProject(id).then(response => {
-      dispatch(resetProjectHint());
-      dispatch(requestRoute(["/origins", origin]));
       dispatch(addNotification({
-        title: "Project deleted",
+        title: "Plan link deleted",
```

Comment:
Ah I see now we are calling it 'plan connection' below... so maybe link wasn't the correct suggestion! :(
Anyways, there's a bunch of other places that we should adopt the  same terminology so that we are consistent - basically everywhere we are replacing 'project' with 'plan'.